Code/Editor/EditorPreferencesPageViewportManipulator.h

```diff
@@ -41,6 +41,16 @@ class CEditorPreferencesPage_ViewportManipulator : public IPreferencesPage
 
         float m_manipulatorLineBoundWidth = 0.0f;
         float m_manipulatorCircleBoundWidth = 0.0f;
+        float m_linearManipulatorAxisLength = 0.0f;
+        float m_planarManipulatorAxisLength = 0.0f;
+        float m_surfaceManipulatorRadius = 0.0f;
+        float m_surfaceManipulatorOpacity = 0.0f;
+        float m_linearManipulatorConeLength = 0.0f;
+        float m_linearManipulatorConeRadius = 0.0f;
+        float m_scaleManipulatorBoxHalfExtent = 0.0f;
+        float m_rotationManipulatorRadius = 0.0f;
+        float m_manipulatorViewBaseScale = 0.0f;
+        bool m_flipManipulatorAxesTowardsView = false;
```

Comment:
Shouldn't these be set to reasonable defaults, by default?

Reply:
These intentionally should not as they're only used to have things appear in the UI (using the `EditContext`). The default values come from the API calls in `ViewportSetttings.cpp`